### DISCUSSION TITLE
Ajuda - Curso de Refatoracão de Testes

### DISCUSSION THREAD

**Discussion by gabrieladouradof** [balta-io/forum#674] [Tira Dúvidas] [answered]
Olá pessoa! Eu finalizei o curso de testes, entretanto no momento de testas as entidades, estou possuindo problemas, apenas a herança "Notifiable" na entidade Entity não é o suficiente, resulta nesse erro: 

![image](https://github.com/user-attachments/assets/1ad4be92-3a67-4433-b8ee-2d627d97143d)

entretanto, eu nao sei qual tipo utilizar, já testei alguns modelos e reli a documentação, mas ainda não interpretei o que posso estar errando. segue a classe Entity: 
![classentity](https://github.com/user-attachments/assets/c3d1b7bc-6134-4483-bb59-6986af21e1a2)

**Comment by henriqueweege** [comment, marked as answer]
Oi, @gabrieladouradof ! Tudo certinho?

Para trazer mais flexibilidade, hoje o Flunt permite que você utilize uma classe customizada de notificação. Para resolver o teu problema, basta herdar de Notifiable < Notification > , ou implementar uma classe customizada para notificação utilizar ela.

![image](https://github.com/user-attachments/assets/ad096c05-a151-434e-87a9-e7ecdcee8c16)


Caso queira, pode colocar a versão do Flunt para a mesma do curso, 1.0.3, que vai funcionar também!

Valeeeeu!

**Reply by gabrieladouradof** [reply]
Olá, Henrique! Muito obrigada pela ajuda. Deu certo.
